### PR TITLE
clarifications and mods for replicability

### PR DESCRIPTION
**Pull request by juliogomez**
include clarifications to make it work in a brand new installation, so that readers can replicate easily

### CODE REVIEW

File: network-services/README.md [hunk 1]
```diff
@@ -5,28 +5,39 @@ This demo shows how model driven programmability can offer even better experienc
...
+<p align="center"> 
+<img src="demo_resources/evpn-fabric.jpg">
+</p>
 
-## Demo Preperation 
-In order to run this demo you'll need to have installed Cisco NSO on your workstation.  You can download and find installation instructions for free up on [DevNet: NSO Getting Started](https://developer.cisco.com/docs/nso/#!getting-nso).  You will need to have either Linux or MacOS platforms to install NSO.  
+## Demo Preparation 
+In order to run this demo you will need to have Cisco NSO installed on your workstation.  You can download the latest NSO version and find installation instructions for free up on [DevNet: NSO Getting Started](https://developer.cisco.com/docs/nso/#!getting-nso).  You will need to have either Linux or MacOS platforms to install NSO. 
 
-> Note: The creation and value of "network services" can be done leveraging multiple tools, both open source and commercial.  Cisco NSO is used in this example as an example platform for network service based configuration management.
```
Comment (hpreston) [inline network-services/README.md]: Why did you remove this Note?

File: network-services/README.md [hunk 2]
```diff
@@ -5,28 +5,39 @@ This demo shows how model driven programmability can offer even better experienc
...
+## Demo Preparation 
+In order to run this demo you will need to have Cisco NSO installed on your workstation.  You can download the latest NSO version and find installation instructions for free up on [DevNet: NSO Getting Started](https://developer.cisco.com/docs/nso/#!getting-nso).  You will need to have either Linux or MacOS platforms to install NSO. 
 
-> Note: The creation and value of "network services" can be done leveraging multiple tools, both open source and commercial.  Cisco NSO is used in this example as an example platform for network service based configuration management.  
+Your NSO version needs to be 4.6 or higher. For reference, my installed NSO version is 4.7.6 and you may download it from [here](https://software.cisco.com/download/home/286321297/type/286283941/release/4.7.6) (please make sure to reboot your workstation after installation).
 
-There are two flavors for this demonstration, `dev` and `test`.
```
Comment (hpreston) [inline network-services/README.md]: Again, why remove this comment?

File: network-services/README.md [hunk 3]
```diff
@@ -222,15 +233,26 @@ The goal of the first part of the demo is to show the current state without netw
             +--rw interface?   string
     ```
 
-1. Now let's use the model to configure the fabric.  
-1. Jump back into NSO and enter config mode. 
+1. Now we need to load the VXLAN-EVPN package into NSO. But before loading a package the YANG data module has to be compiled into _fxs_ files that can be loaded into NSO.
+
+    ```
+    make -C packages/vxlan-evpn/src
```
Comment (hpreston) [inline network-services/README.md]: What was the reason you needed to do this? I didn't in my example and would prefer that we just include the compiled FXS files in the repo rather than make people compile themselves.

Comment (juliogomez) [reply network-services/README.md]: The included VXLAN FXS files from the repo were not working for me, hence my need to recompile them.
